storage/aptosdb/src/lib.rs

```diff
@@ -1284,7 +1284,7 @@ impl DbWriter for AptosDB {
                 version,
                 base_version,
             )?;
-            debug!(
```

Comment:
Do we have debug logs enabled ?

Reply:
In most of our networks we do (e.g., AIT, devnet, etc.). Non-aptos nodes can configure it, but it seems like we're defaulting to debug for the time being. 'cc @sherry-x ?